src/instructlab/config.py

```diff
@@ -190,3 +192,7 @@ def write_config(cfg, config_file=DEFAULT_CONFIG):
 def get_api_base(host_port):
     """Returns server API URL, based on the provided host_port"""
     return f"http://{host_port}/v1"
+
+
+def get_model_family(forced, model_path):
+    return forced if forced else match(r"^\w*", path.basename(model_path)).group(0)
```

Comment:
It looks like `match()` can return `None`, but that isn't handled here

Reply:
How can it return `None`?

Reply:
https://docs.python.org/3/library/re.html#re.match

Reply:
`^\w*` will return at least an empty string. In will not return `None`.

Reply:
Ah, right. I didn't read the regex itself closely enough. Thanks.